src/lib/yang/binary.lua

```diff
@@ -210,11 +210,19 @@ local function data_emitter(production)
          end
       end
    end
+   local native_types = {
+      ["instance-identifier"] = true,
+      empty = true,
+      enumeration = true,
+      identityref = true,
+      leafref = true,
```

Comment:
In current master, `leafref` is not implemented yet. I can remove `leafref` from the list and add it later once `leafref` is mplemented. There's an attempt to implement `leafref` in https://github.com/Igalia/snabb/pull/827. It works but I think the code needs to be refactored as it's not very elegant.